### ISSUE TITLE
Replace project selector hover-over with checkbox

### ISSUE THREAD

**Issue by theathorn** [DataBiosphere/data-browser#415]
Beta feedback indicates that users would prefer a Project checkbox instead of the current "+Select Project" hover over.
Perhaps this can be added to the left of the Project Name text?

┆Issue Number: HCAB-346

**Comment by theathorn** [comment]
I agree with @adriennes : Instead of having all other projects disappear from the list once an item is selected, could we keep the view the same, and allow a user to select multiple items?

For example, if you go to "More" and select multiple projects there, then those show up in the Projects view with checkboxes selected so the current behavior seems inconsistent.

**Comment by NoopDog** [comment]
Yes, I agree as well and we have been working on mocks to that effect.  I just realized that it may be easier to implement this than I thought without much change to the existing system.  So will look at doing that. 

Meanwhile, this ticket "Replace project selector hover-over with checkbox" I believe can be closed and  we can track "Don't restrict project tab by project" as a separate ticket. 

Once this is in place i think we can remove the project facet from the drop downs above. I have created https://github.com/HumanCellAtlas/data-browser/issues/419 to track the "Don't restrict project tab by the selected project".
